code/Sectoral chapters panel 1/code/0 Data prep.R

```diff
@@ -10,11 +10,13 @@ source('0 report production/GTA 25/help files/GTA 25 cutoff and definitions.R')
 
 this.chapter=c(paste0("Sectoral chapters - Sector ",paste0(sectors)))
 
-wdpath = "0 dev/gta-25/code/Sectoral chapters panel 1/"
+wdpath = "0 dev/gta-25-pb/code/Sectoral chapters panel 1/"
 data.path = paste0(wdpath,"data/")
 
 run.calc=T
 
+trade.data.year = 2018
```

Comment:
please confirm that SE asked for trade shares calculated on 2018 basis, not usual GTA base (2005-07)